client/my-sites/marketing/connections/service.jsx

```diff
@@ -459,7 +460,12 @@ export class SharingService extends Component {
 	}
 
 	shouldBeExpanded( status ) {
-		if ( this.isMailchimpService() && this.state.justConnected ) {
+		if (
+			this.isMailchimpService() &&
+			( this.state.justConnected ||
+				( this.props.queryArgs && 'mailchimp' in this.props.queryArgs ) )
+		) {
+			window.scrollTo( 0, 9999 );
```

Comment:
Why `9999`?

Would this work as well?
```js
if ( typeof window !== 'undefined' ) {
   window.scrollTo( 0, 0 );
}
```

Reply:
Wouldn't that take people to the top of the screen? The idea is for them to be taken to the bottom, because that's where the Mailchimp setting is. 9999 is a completely random number for that though. :)